runtime/README.md

```diff
@@ -312,6 +316,64 @@ Table: `INVOKE_DPE_COMMAND` output arguments
 | data_size   | u32           | Length in bytes of the valid data in the data field
 | data        | u8[...]       | DPE response structure as defined in the DPE iRoT profile.
 
+### QUOTE\_PCRS
+
+Generate a signed quote over all Caliptra hardware PCRs using the Caliptra PCR quoting key.
+All PCR values are hashed together with the nonce to produce the quote.
```

Comment:
Would eventually be good to update this with the exact formula for validating the signature.

Reply:
Ack. I'll update this once the implementation is done and I can verify how it's done.